src/components/NotKevButton.js

```diff
@@ -1,21 +1,28 @@
 import '../css/App.css';
 
-const NotKevButton = ({currentAuthorProp, authorLabelProp, setGameStateProp, redButtonProp, setRedButtonProp}) => {
+const NotKevButton = ({statusProp, setStatusProp, authorProp, pointsProp, setPointsProp}) => {
         
-    const handleClick = () => { 
-            if (currentAuthorProp === "Kevin McCloud") 
-                { setRedButtonProp("WRONG!");
-                setGameStateProp("incorrect");
-            } else {
-                setRedButtonProp("CORRECT!");
-                authorLabelProp.textContent = `${currentAuthorProp}`;
-                setGameStateProp("correct");
+    let buttonText = "Not Kev";
+
+    if (statusProp === "correct-notkev") {
```

Comment:
So I'm not replicating comments, all the comments in ItsKev are applicable in here too in my view